src/io/flutter/run/bazelTest/BazelTestFields.java

```diff
@@ -179,11 +191,19 @@ else if (!bazelTarget.startsWith("//")) {
     }
   }
 
+  /**
+   * Starts running the tests.
+   */
+  ProcessHandler run(Project project, RunMode mode) throws ExecutionException {
```

Comment:
Consider annotating project and mode as `@NotNull`?  It looks like `getLaunchCommand(..)` expects that...

Reply:
I'll follow up with a PR.

Reply:
https://github.com/flutter/flutter-intellij/pull/1937